adding_species_manuscript.md

```diff
@@ -389,31 +391,238 @@ Table of species/demographic scenarios added/worked on by community since origin
+
+```
+python -m maintenance add-species Anopheles_gambiae
+```
+
+A partial list of the genomes housed on Ensembl [can be found here](https://metazoa.ensembl.org/species.html).
+
+The next steps are going to be to flesh out the code templates that the maintenance code just wrote for us in our branch. 
+
+The function call above has created a few new files in the directory structure of stdpopsim, in particular at the level of ```stdpopsim/catalog``` this utility has created a new directory called AnoGam, which contains three files. The 3 + 3 naming scheme for this directory (and all following references in stdpopsim) is automatically parsed by the maintenance script:
+
+```
+├── catalog
+│   ├── AnoGam
+│   │   ├── __init__.py
+│   │   ├── genome_data.py
+│   │   └── species.py
+```
+* The ```__init__.py``` contains information about XXXXXXXXXX 
+* The ```genome_data.py``` contains a data dictionary which has slots for the assembly accession number, the assembly name, and a dict representing the chromosome names and their associated lengths.
+* The ```species.py``` is the file that we will need to edit with our species specific information. The basics that we will need here include:
+  * 1) *chromosome-specific* recombination rates (could also be single rate)
+  * 2) a genome-wide average *mutation rate*
+  * 3) a *generation time* estimate (in years)  
+  * 4) a default *population size*
+  * 5) literature citations that one can point to for the above and the assembly
+  
+- Test the code while building the files
+
+```stdpopsim``` in part guarantees code quality through the use of unit testing. Basic sanity tests for this new species will be completed through QC over in the ```tests/test_AnoGam.py``` file created by the command ```python -m maintenance add-species Anopheles_gambiae``` above.
+
+To run the tests in stdpopsim uses the pytest module. We will quickly run the tests that our maintenance function has stubbed out and expect it to error out:
+
+```
+python -m pytest tests/test_AnoGam.py
+```
+The test will tell that the required information entered into this species definition is missing.
+Let's add it.
+
+### 1) Recombination rates
+As a reference for recombination rates in *Anopheles gambiae* we use a recombination map based on a study from Pombi et al. (2006). In that manuscript the authors cite rates around 1cM/Mb, with a bit of variation among arms. In particular let's edit the following block of code in species.py that defines the recombination_rate dict:
+
+```
+_recombination_rate = {
+    "2L": 0, #setting to zero because of inversion 
+    "2R": 1.3e-8, 
+    "3L": 1.6e-8, 
+    "3R": 1.3e-8, 
+    "X": 1e-8,
+    "Mt": 0
+}
+```
+Every information in regards to the population genetics parameters requires a citeable reference. To do that we can create a stdpopsim.Citation object to the same species.py file. that Citation object looks like this:
```

Comment:
"Every piece of information requires a citable reference."

Reply:
done